tox.ini

```diff
@@ -0,0 +1,66 @@
+[tox]
+envlist = py{35}-django{22},py{38}-django{22}
+
+[doc8]
+max-line-length = 120
+
+[pycodestyle]
+exclude = .git,.tox,migrations
+max-line-length = 120
+
+[pydocstyle]
+; D101 = Missing docstring in public class
+; D200 = One-line docstring should fit on one line with quotes
+; D203 = 1 blank line required before class docstring
+; D212 = Multi-line docstring summary should start at the first line
+; D215 = Section underline is over-indented (numpy style)
+; D404 = First word of the docstring should not be This (numpy style)
+; D405 = Section name should be properly capitalized (numpy style)
+; D406 = Section name should end with a newline (numpy style)
+; D407 = Missing dashed underline after section (numpy style)
+; D408 = Section underline should be in the line following the section’s name (numpy style)
+; D409 = Section underline should match the length of its name (numpy style)
+; D410 = Missing blank line after section (numpy style)
+; D411 = Missing blank line before section (numpy style)
+; D412 = No blank lines allowed between a section header and its content (numpy style)
+; D413 = Missing blank line after last section (numpy style)
+; D414 = Section has no content (numpy style)
+ignore = D101,D200,D203,D212,D215,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414,D401
+match-dir = (?!migrations)
+
+[pytest]
+DJANGO_SETTINGS_MODULE = test_settings
+addopts = --cov enable_retired_username_email --cov-report term-missing --cov-report xml
+norecursedirs = .* docs requirements
+
+[testenv]
+deps =
+    django22: Django>=2.2,<2.3
+    -r{toxinidir}/requirements/test.txt
+commands =
+    pytest {posargs}
+
+[testenv:quality]
+whitelist_externals =
+    make
+    rm
+    touch
+deps =
+    -r{toxinidir}/requirements/quality.txt
+commands =
+    touch tests/__init__.py
```

Comment:
can we create an empty __init__.py file?

Reply:
I am not sure what you mean here. Create `init.py` where ?

`touch tests/__init__.py` already creates an empty `init.py` file. Isn't it ?

Reply:
I don't understand why are ew doing this :)

Reply:
Sorry .. I misunderstood your question :)

This piece of code is from the [custom-student-verification](https://github.com/open-craft/custom-student-verification/) app which is used as a template for the current app.

I quickly looked into it and apparently pylint looks for the `__init__.py` file to be compliant with PEP specifications. To verify the same, I removed this command and ran the quality checks in devstack and pylint threw an error :

![pylint_error](https://user-images.githubusercontent.com/36229857/151688507-5431e6d2-8320-408f-9f93-c30a29288d11.png)

As to why isn't there a permanent `__init__.py` file and the file is created and deleted everytime the quality checks run, my guess is that the `tests` directory is in not intended to be a package and only a container for test files.